Lab 1/README.md

```diff
@@ -2,18 +2,28 @@
 
 Learn how to push an image of an application to IBM Cloud Container Registry and deploy a basic application to a cluster.
 
-
-# 1. Push an image to IBM Cloud Container Registry
+# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
 
 If you haven't already:
 1. Install the CLIs and Docker, as described in the "Prerequesite" section.  
 2. Provision a cluster: 
 
    ```bx cs cluster-create --name <name-of-cluster>```
 
-To push an image:
+# 1. Push an image to IBM Cloud Container Registry
+
+To push an image, we first must have an image to push. We have
+prepared several `Dockerfile`s in this repository that will create the
+images. We will be running the images, and creating new ones, in the
+later labs. 
+
+This lab uses the Container Registry built in to IBM Cloud, but the
+image can be created and uploaded to any standard Docker registry to
+which your cluster has access.
+
+1. Download a copy of this repository:
 
-1. Download from https://github.com/IBM/container-service-getting-started-wt
+    ```git clone https://github.com/IBM/container-service-getting-started-wt```
```

Comment:
I don't know what "Download from" means. I'm assuming we want them to clone the repo, and assuming they know what that means.